xml/libvirt_networking.xml

```diff
@@ -8,958 +8,957 @@
  <title>Managing Networks</title>
  <info>
   <abstract>
-   <para>
-    This chapter introduces common networking configurations supported by
-    &libvirt;. It does not depend on the hypervisor used. It is valid for all
-    hypervisors supported by &libvirt;, such as &kvm; or &xen;. These setups
-    can be achieved using both the graphical interface of &vmm; and the command
-    line tool <command>virsh</command>.
+    <para>
+     This chapter describes common network configurations for a &vmhost;,
+     including those supported natively by the &vmhost; and &libvirt;.
+     The configurations are valid for all hypervisors supported by &productname;,
+     such as &kvm; or &xen;.
    </para>
   </abstract>
   <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
    <dm:bugtracker></dm:bugtracker>
   </dm:docmanager>
  </info>
  <para>
-  There are two common network setups to provide a &vmguest; with a network
-  connection:
+  There are two common network configurations to provide a &vmguest; with a
+  network connection:
```

Comment:
a guest network connection from where to where?

Reply:
I don't think "where to where" is necessary. We don't generally say "I need a network connection from my server's ethernet port to a switch". It is enough to say I need a network connection for my server, no?

Reply:
Hi Jim, I don't think that from a user perspective it is clear what kind of use case each network configuration allows, and which limitations it has.

As an example, with out of the box upstream qemu user networking, there are a lot of limitations for the VM.

The VM can initiate a connection to the external network, but the VM is not reachable from the external connections, so it can not run a reachable service this way. Also ICMP is limited etc.

With these two network configurations, what are, if any, the limitations for the user? Is the VM going to be visible externally, and how? Any limitations to the traffic type? Any performance consideration of the one versus the other for the user?

Reply:
For inspiration we could look at something like:
https://jamielinux.com/docs/libvirt-networking-handbook/

We do not need to include all options like there, just the two we care about and support.

But rather than an in-detail description of everything under the sun (which can still come later), the entry point for the user could be: how do I choose the right networking for me? What will it allow me to do, and what will it instead be limited? Does it fit my requirements?

Reply:
I added a bit more content to the generic description of bridged network vs libvirt virtual network and will force push shortly. E.g. I mentioned that layer 3 forwarding "introduces additional packet processing on the &vmhost; as compared to a Layer 2 bridged network." IMO this PR is a huge improvement over the existing text. Further improvements can be explored when/if we have a "doc day".